provisioning/service/tests/Config/IndividualEnrollmentTests.cs

```diff
@@ -266,6 +270,7 @@ public void IndividualEnrollmentConstructorWithoutCapabilitiesJSONSucceed()
             Assert.IsTrue(individualEnrollment.Attestation is X509Attestation);
             Assert.AreEqual(SampleDeviceId, individualEnrollment.DeviceId);
             Assert.AreEqual(SampleIotHubHostName, individualEnrollment.IotHubHostName);
+            Assert.IsNotNull(individualEnrollment.OptionalDeviceInformation);
```

Comment:
Shall we confirm the contents of the payload as well?

Reply:
Done